image/tarexport/load.go

```diff
@@ -77,6 +78,9 @@ func (l *tarexporter) Load(inTar io.ReadCloser, outStream io.Writer, quiet bool)
 		if err != nil {
 			return err
 		}
+		if img.OS != runtime.GOOS {
+			return fmt.Errorf("cannot load %s image on %s", img.OS, runtime.GOOS)
```

Comment:
Thanks! Yeah, I actually thought of that case, but had not addressed it yet. I'm not sure about other cases, but will look into it.